src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1378,7 +1378,7 @@ internal bool FilterAndAppendDiagnostics(DiagnosticBag accumulator, IEnumerable<
                 {
                     continue;
                 }
-                else if (filtered.Severity == DiagnosticSeverity.Error &&
+                else if (filtered.DefaultSeverity == DiagnosticSeverity.Error &&
                     !filtered.IsSuppressed)
```

Comment:
>!filtered.IsSuppressed [](start = 20, length = 22)

Do we still have to check this?

Reply:
Yes, for the same reason as in the above comment.

---
In reply to: [295480408](https://github.com/dotnet/roslyn/pull/36582#discussion_r295480408) [](ancestors = 295480408)